### DISCUSSION TITLE
text-wrap class or controlling width with long string of characters

### DISCUSSION THREAD

**Discussion by colinambler** [bootstrap-email/bootstrap-email#216] [Help]
Has anyone encountered a work-around for applying the standard bootstrap ["text-wrap" class ](https://getbootstrap.com/docs/4.4/utilities/text/#word-break)?  I have an instance where I'm sharing a product key via email, but it's overwhelming the width of the email (see screenshot).  I haven't been able to find a way to make it work...

<img width="1420" alt="Screen Shot 2022-12-21 at 11 35 50 AM" src="https://user-images.githubusercontent.com/10900620/208957045-4d7317a2-a28e-4e48-87c7-f63e28bfe08f.png">

**Comment by stuyam** [comment]
Yeah this would be a good one to add.
